src/ui/components/ToggleOnMobileCard/styles.scss

```diff
@@ -0,0 +1,61 @@
+@import "~photon-colors/colors";
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+
+.ToggleOnMobileCard-ToggleLink {
+  display: flex;
+  justify-content: space-between;
+  text-decoration: none;
+
+  &:link {
+    color: $grey-60;
+  }
+
+  @include respond-to(large) {
+    cursor: default;
+  }
+}
+
+.ToggleOnMobileCard-ToggleArrow {
```

Comment:
Is this supposed to render an arrow in the header to indicate that tapping on the header will expand it? I don't see an arrow:

<img width="322" alt="screenshot 2017-08-28 15 53 43" src="https://user-images.githubusercontent.com/55398/29794413-812a82fe-8c0d-11e7-80aa-a28d301691f5.png">